### ISSUE TITLE
protect bill by vote id view from failing on missing vote_id

### ISSUE THREAD

**Issue by alonisser** [hasadna/Open-Knesset#690]
how is this possible
[original_error])https://opbeat.com/hasadna/openknesset-prod/errors/181/)

I believe we should add input validations..

**Comment by itaibs** [comment]
@alonisser The link directs to a 404.

**Comment by alonisser** [comment]
Yeah, they don't keep the data that long, In recent bugs, I copy the stacktrace from there, but the bug is still relevant,  validating input in this view. Please note that a PR for this kind of bug, should be accompanied with a test
